tests/e2e/utils/nav-menu-utils.js

```diff
@@ -1,17 +1,14 @@
 /**
  * WordPress dependencies
  */
-import {
-	createMenu,
-	deleteAllMenus,
-	visitAdminPage,
-} from '@wordpress/e2e-test-utils';
+import { createMenu, deleteAllMenus } from '@wordpress/e2e-test-utils';
 
-export async function createTestMenu() {
+export async function createTestMenu(menuLocation = 'top') {
 	await deleteAllMenus();
 	await createMenu(
 		{
 			name: 'Test Menu 1',
+			locations: [menuLocation],
```

Comment:
So this makes the `assignMenuToLocation` function obsolete? It was doing additional work that wasn't needed?